pynestml/codegeneration/resources_nest/point_neuron/common/SynapseHeader.h.jinja2

```diff
@@ -286,6 +278,69 @@ private:
     State_() {};
   };
 
+  /**
+   * Free parameters of the synapse.
+   *
+   {{synapse.print_parameter_comment("*")}}
+   *
+   * These are the parameters that can be set by the user through @c SetStatus.
+  * Parameters do not change during calls to ``send()`` and are not reset by
+  * @c ResetNetwork.
+  *
+  * @note Parameters_ need neither copy constructor nor @c operator=(), since
+  *       all its members are copied properly by the default copy constructor
+  *       and assignment operator. Important:
+  *       - If Parameters_ contained @c Time members, you need to define the
+  *         assignment operator to recalibrate all members of type @c Time . You
+  *         may also want to define the assignment operator.
+  *       - If Parameters_ contained members that cannot copy themselves, such
+  *         as C-style arrays, you need to define the copy constructor and
+  *         assignment operator to copy those members.
+  */
+  struct Parameters_{
+{%- filter indent(4,True) %}
+{%- for variable in synapse.get_parameter_symbols() %}
+{%-     set isHomogeneous = PyNestMLLexer["DECORATOR_HOMOGENEOUS"] in variable.get_decorators() %}
+{%-     if (not isHomogeneous) %}
+{%-   include 'directives/MemberDeclaration.jinja2' %}
+{%-     else %}
+    // N.B. the parameter `{{names.name(variable)}}` is defined in the common properties class
+{%-     endif %}
+{%- endfor %}
+{%- endfilter %}
+
+{% if uses_numeric_solver %}
+    double __gsl_error_tol;
+{% endif %}
+
+    /** Initialize parameters to their default values. */
+    Parameters_() {};
+  };
+
+  /**
+   * Internal variables of the synapse.
+   *
+   {{synapse.print_internal_comment('*')}}
+   *
+   * These variables must be initialized by @c pre_run_hook (or calibrate in NEST 2.x), which is called before
```

Comment:
Should we remove this comment as the synapse doesn't have a `calibrate()` function?